bitnami/kibana/values.yaml

```diff
@@ -14,7 +14,7 @@
 image:
   registry: docker.io
   repository: bitnami/kibana
-  tag: 7.12.1-debian-10-r0
+  tag: 7.12.1-debian-10-r24
```

Comment:
Could you please revert this change?

The bitnami-bot will take care of updating the image tag with the latest changes before merging this PR.

Reply:
Done.   Automation is great!